m2cgen/interpreters/visual_basic/linear_algebra.bas

```diff
@@ -0,0 +1,26 @@
+Function addVectors(ByRef v1() As Double, ByRef v2() As Double) As Double()
```

Comment:
I have no knowledge about VB whatsoever so forgive me my ignorance: why do static files have extension `bas` but generated ones - `vb`?

Reply:
`bas` is a file extension of pure BASIC language. I used it here to show that these files are both VBA and VB compatible. `vb` is a file extension for modern VB language, for which examples are generated.